File: articles/features-labels-and-the-politics-of-measurement/cpp/measurement_review.cpp
#include <iostream>
#include <iomanip>
double rate(double num, double den) { return den == 0.0 ? 0.0 : num / den; }
int main() { std::cout << std::fixed << std::setprecision(4) << "sensitivity=" << rate(72, 94) << "\n"; }

File: articles/features-labels-and-the-politics-of-measurement/java/MeasurementReview.java
public class MeasurementReview {
  static double rate(double num, double den) { return den == 0.0 ? 0.0 : num / den; }
  public static void main(String[] args) { System.out.printf("specificity=%.4f%n", rate(88, 106)); }
}
